### PR TITLE
ENH Replace ADMIN permissions with less permissions in Behat test

### PR DESCRIPTION
**Pull request by sabina-talipova**
### Description
Replace user with "Admin" permissions to user with "Editor" permissions.

### Parent issue
- https://github.com/silverstripe/silverstripe-behat-extension/issues/220

### CODE REVIEW

File: tests/behat/features/manage-campaigns.feature [hunk 1]
```diff
@@ -17,7 +18,7 @@ Feature: Manage campaigns
       And I press the "Create" button
     Then the "p.alert" element should contain "Nice one! You have successfully created a campaign."
       And the "Publish campaign" button should be disabled
-
+
```
Comment (GuySartorelli) [inline tests/behat/features/manage-campaigns.feature]:
```suggestion
```

File: tests/behat/features/manage-campaigns.feature [hunk 2]
```diff
@@ -6,7 +6,8 @@ Feature: Manage campaigns
 
   Background:
     Given a "campaign" "Test Campaign" with "Description"="this is a test"
-      And I am logged in with "ADMIN" permissions
+      And the "group" "EDITOR" has permissions "Access to 'Pages' section" and "Access to 'Campaigns' section" and "Access to 'Files' section" and "FILE_EDIT_ALL"
```
Comment (GuySartorelli) [inline tests/behat/features/manage-campaigns.feature:9]: Are _all_ of these permissions required for _all_ of these scenarios? Seems likely that some of these can be split out to use less permissions.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: tests/behat/features/manage-campaigns.feature (hunk 1 above)
Comment (sabina-talipova) [reply tests/behat/features/manage-campaigns.feature]: DONE

File: tests/behat/features/manage-campaigns.feature (hunk 2 above)
Comment (sabina-talipova) [reply tests/behat/features/manage-campaigns.feature:9]: I created additional user with only "Campaign section" permissions.
But for the last `Scenario: Publishing a campaign publishes associated assets` I left "EDITOR" with permission as is.

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**